4.6.2/WcfBasicIISHost/WcfServiceTestClient/Service References/ServiceReference1/Reference.svcmap

```diff
@@ -18,14 +18,14 @@
     <ServiceContractMappings />
   </ClientOptions>
   <MetadataSources>
-    <MetadataSource Address="http://localhost/wcfservicetest/Service1.svc" Protocol="http" SourceId="1" />
+    <MetadataSource Address="http://10.0.75.1:84/Service1.svc" Protocol="http" SourceId="1" />
```

Comment:
All the code under servicereference should be regenerated on customer machine, I probably should remove all the files there.